### DISCUSSION TITLE
Is Instagram canceling the download from their end or could this be a bug in the crawler?

### DISCUSSION THREAD

**Discussion by Aston77** [AAndyProgram/SCrawler#172] [Q&A]
I've noticed odd behavior with Instagram lately where SCrawler (with up-to-date cookies, **x-csrftoken** & **x-ig-www-claim**) will start downloading a small/medium size group of profiles and then breaks.

Lately, x-csrftoken doesn't change and generally matches the browser (for a while it changed often). 

The most common things that occur...

- **x-ig-www-claim** does change on occasion, but it's become fairly common for the value to be zero'd out in the settings if the download breaks
- Another thing that happens is the **Download Timeline** & **Download Stories** buttons start as checked and often one or both will be unchecked if the download breaks.

FWIW the "# of spent requests" value does often go up initially when it is working, but when the download breaks the setting will be "0" when you return to the settings window.

There isn't a great deal of data in the logs

![image](https://github.com/AAndyProgram/SCrawler/assets/78501124/30da155c-5e87-4d6e-b0af-639916139bb6)

**Comment by AAndyProgram** [comment]
Example profile URL?
